src/main/resources/openapi/transcriptions.yaml

```diff
@@ -60,8 +60,43 @@ paths:
                         $ref: '#/components/schemas/RequestTranscription400ErrorCode'
         '401':
           description: Unauthorised Error
+          content:
+            application/json+problem:
+              schema:
+                $ref: './problem.yaml'
+              example:
+                type: "AUTHORISATION_100"
+                title: "User is not authorised for the associated courthouse"
+                status: 401
```

Comment:
I think this problem / example should be moved under 403 Forbidden Error - it looks to have been missed when @karen-hedges added the `@Authorisation` to the TranscriptionController

Reply:
Removed

Reply:
Re-added back in the correct place.